src/fsharp/SimulatedMSBuildReferenceResolver.fs

```diff
@@ -0,0 +1,229 @@
+// Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+#if INTERACTIVE
+#load "../utils/ResizeArray.fs" "../absil/illib.fs" "../fsharp/ReferenceResolver.fs"
+#else
+module internal Microsoft.FSharp.Compiler.SimulatedMSBuildReferenceResolver 
+#endif
+
+open System
+open System.IO
+open System.Reflection
+open Microsoft.Win32
+open Microsoft.FSharp.Compiler
+open Microsoft.FSharp.Compiler.ReferenceResolver
+open Microsoft.FSharp.Compiler.AbstractIL.Internal.Library
+
+let internal SimulatedMSBuildResolver =
```

Comment:
Could this be exposed? I'm trying to remove the ProjectCracker dependency using Forge.ProjectSystem but I need a way to resolve dependencies. This would be very useful to resolve the core dependencies of .fsproj file.